scripts/contentScript.js

```diff
@@ -138,10 +138,17 @@ async function sha256(value) {
 }
 
 // for showing all tracked/ untrack pr in a organization
-async function getTrackedRepos(orgName, userId) {
+async function getTrackedRepos(orgName, userId, is_github) {
```

Comment:
I think the `getTrackedRepos` function is currently only called for the Github case at the moment. 
@ajayVibinex Please confirm.

Reply:
Yes, @avikalpg its for gitHub only. basically this is for , when the loading finishes inside a repo so to check weather it's tracked or not. We can also add it to bitbucket.

Reply:
So I am not supposed to remove the function right?

Reply:
Yes don't remove this function it's important for showing the reload and add button !